src/Link.php

```diff
@@ -312,6 +312,46 @@ public static function generateLinkContents($link, CommonDBTM $item/*, bool $saf
         ) {
             $link = str_replace("[SERIAL]", $item->getField('serial'), $link);
         }
+	if (
+	    strstr($link,"[MODEL]")
+	    && $item->isField('computermodels_id')
+	) {
+            $link = str_replace("[MODEL]",
+		    str_replace(" ", "_", Dropdown::getDropdownName("glpi_computermodels", $item->getField('computermodels_id'))),
```

Comment:
What is the purpose of this `str_replace(" ", "_", ...)` ?

Reply:
Thanks Cedric, this str_replace(" ",_) can be removed. It is how a client preferred the handling of spaces, but I recognize now that it is not general.

Reply:
You should indeed remove it.